platforms/macosx/src/GSystemOSX.cpp

```diff
@@ -108,7 +108,46 @@ VGDevice* GSystemOSX::CreatePrinterDevice( )
 // --------------------------------------------------------------
 const VGFont* GSystemOSX::CreateVGFont( const char * faceName, int size, int properties ) const
 {
-	return faceName ? new GFontOSX (faceName, size, properties) : 0;
+    if (faceName == NULL) {
```

Comment:
why all this stuff here and not in GFontOSX ? I think that could produce different behaviours, depending on whether the font is allocated directly or using GSystem

Reply:
Thanks for this comment. Moving to GFontOSX makes it simpler. The only caveat then is, that GFontOSX::GetName() may be different from the actually used font name. In the previous implementation, that name was used in CGContextSelectFont. But that is dropped, so we should be fine.